bin/antigen.zsh

```diff
@@ -117,13 +117,11 @@ antigen () {
 }
 
 -antigen-get-clone-url () {
-  # Takes a repo's clone dir and gives out the repo's original url that was
-  # used to create the given directory path.
+  # Takes a repo's clone dir and unmangles it, to give the repo's original url
+  # that was used to create the given directory path.
 
   if [[ "$1" == ".zprezto" ]]; then
-    # Prezto's (in `.zprezto`), is assumed to be from `sorin-ionescu`'s
-    # remote.
-    echo https://github.com/sorin-ionescu/prezto.git
+    echo "$(cd "$ADOTDIR/repos/.zprezto" && git config --get remote.origin.url)"
```

Comment:
This is used to generate output for the user. Given this is now configurable, it seems better to check which repo was used.